src/room.js

```diff
@@ -0,0 +1,37 @@
+class Room {
+
+  constructor (roomname) {
+    this.roomname = roomname;
+    this.endpoints = {};
+  }
+  addEndpoint (endpointId) {
+    this.endpoints[endpointId] = {
+      name: '',
+      permissions: 'CHAT',
+      commsid: ''
+    };
+  }
+  updateEndpointName (endpointId, Name) {
+    this.endpoints[endpointId].name = Name;
```

Comment:
+ this is not the "endpoint name", this is the user's username. Can we change this variable to `username` @njsfield?
+ any reason why the parameter `Name` starts with a capital letter?